server/app/views/admin/programs/ProgramImageView.java

```diff
@@ -32,7 +44,45 @@ public Content render(Http.Request request, ProgramDefinition programDefinition)
         String.format(
             "Manage program image for %s", programDefinition.localizedName().getDefault());
     H1Tag headerDiv = renderHeader(title, "my-10", "mx-10");
-    HtmlBundle htmlBundle = layout.getBundle(request).setTitle(title).addMainContent(headerDiv);
+    FormTag imageDescriptionForm = createImageDescriptionForm(request, programDefinition);
+    HtmlBundle htmlBundle =
+        layout.getBundle(request).setTitle(title).addMainContent(headerDiv, imageDescriptionForm);
+
+    // TODO(#5676): This toast code is re-implemented across multiple controllers. Can we write a
+    // helper method for it?
+    Http.Flash flash = request.flash();
+    if (flash.get("error").isPresent()) {
+      htmlBundle.addToastMessages(ToastMessage.errorNonLocalized(flash.get("error").get()));
```

Comment:
question (blocking): Ah, maybe this is error handling since it will only show the success if there are no errors? How/when would this error get set though?

Reply:
Oops, you're right that I never actually send an "error" toast. Removed this `if` check!